apps/babydegen-ui/src/components/WithdrawAgentsFunds/WithdrawAgentsFunds.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button, Card, Flex, Typography } from 'antd';
+import { useState } from 'react';
+
+import { CardTitle } from '../../ui/CardTitle';
+import { COLOR } from '../../constants/colors';
+import { InitiateWithdrawal } from './InitiateWithdrawal';
+
+const { Text } = Typography;
+
+export const WithdrawAgentsFunds = () => {
+  const [canInitiateWithdrawal, setCanInitiateWithdrawal] = useState(false);
+
+  return (
+    <Card className="card-gradient">
+      <Flex vertical justify="space-between" align="flex-start" gap={24}>
+        <Flex vertical gap={8}>
+          <CardTitle
+            text={canInitiateWithdrawal ? 'Withdraw invested funds' : 'Withdraw agents funds'}
```

Comment:
don't understand, why using different titles. it's like there are two steps of withdrawing being on the UI